packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -153,6 +153,7 @@ export type GridConfig<T extends { __typename?: string }> = {
     copyPaste?: boolean;
     readOnly?: boolean;
     initialSort?: Array<{ field: string; sort: GridSortDirection }>;
+    initialFilter?: Omit<GridFilterModel, "linkOperator"> & { linkOperator?: "and" | "or" };
```

Comment:
can you explain the problem with linkOperator here?

Reply:
My goal here was to keep the variables easy to use for devs, paralleling `initialSort` with lowercase values.

There are multiple reasons here:

1. I did not want to introduce another import to the cometGen files.
As far as I understood, these files should not need to import lots of things. That's why I decided to use string values  instead of the `GridLinkOperator` directly.

2.  `GridLinkOperator` is an Enum that expects uppercase values "And" | "Or".
Meaning, the values for `linkOperator` in cometGen files would need to be uppercase and therefore be confusing to use, as all other variables are lowercase.

That's why I decided against using `GridLinkOperator` directly and override the values with the Omit<>.

Reply:
:+1:

you are only supporting items and linkOperator of GridFilterModel, and linkOperator is overridden. Might be better to replace GridFilterModel usage with a custom type (containing only items (=`GridFilterItem[]`) and linkOperator)